pkl-core/src/test/files/LanguageSnippetTests/input/types/trailingCommas.pkl

```diff
@@ -0,0 +1,22 @@
+class Foo<
+  Key,
+  Value,
+> {
```

Comment:
I think let's add a one-off test in `org.pkl.core.parser` for class/method type params, and remove this one so that our expected output isn't "Only standard library members can have type parameters."

And you can add a comment there explaining why the test is implemented as a junit test (see `MultiLineStringLiteralTest`)